glean-core/android/src/main/java/mozilla/telemetry/glean/net/HttpURLConnectionUploader.kt

```diff
@@ -90,7 +90,7 @@ class HttpURLConnectionUploader : PingUploader {
                     // Known other errors:
                     // 500 - internal error
 
-                    // For all other errors we log a warning an try again at a later time.
```

Comment:
If you keep finding and fixing a couple of these with each python PR, you the Kotlin is going to be quite tidy :D